ppr-ui/src/components/common/ButtonFooter.vue

```diff
@@ -199,7 +199,7 @@ export default defineComponent({
       )
     })
     const cancel = () => {
-      if (hasUnsavedChanges.value) localState.showCancelDialog = true
+      if (hasUnsavedChanges.value === true) localState.showCancelDialog = true
```

Comment:
Could be simplified to `if (hasUnsavedChanges.value) ...`

Reply:
I tested and it does not evaluate correct.  if not testing for true, the expression will not evaluate correct.  I have tested